source/features/more-dropdown.tsx

```diff
@@ -32,20 +31,25 @@ async function init() {
 
 	select('.reponav-dropdown .dropdown-menu').append(
 		<a href={`/${repoUrl}/compare`} class="rgh-reponav-more dropdown-item" data-skip-pjax>
-			{icons.darkCompare()}
-			{' Compare'}
+			{icons.darkCompare()} Compare
 		</a>,
 
 		isEnterprise() ? '' :
```

Comment:
@busches is this still necessary?

Reply:
We upgraded this weekend, let me check tomorrow!

Reply:
Still don't have dependencies link in `2.16.4` :/